File: Python/Logic-1/special_eleven.py

# we'll say a number is special if it is a multiple of 11 or if it is one more than a multiple of 11
# return true if the given non-negative number is special
def special_eleven(n):
    b = False

    if n >= 0:
        if n % 11 == 0 or n % 11 == 1:
            b = True

    return b

print(special_eleven(22))
print(special_eleven(23))
print(special_eleven(24))

File: Java/Logic-1/specialEleven.java

public class logic1 {

	// we'll say a number is special if it is a multiple of 11 or if it is one more than a multiple of 11
	// return true if the given non-negative number is special
	public boolean specialEleven(int n) {
		
		boolean b = false;
		
        if (n >= 0) {
            if (n % 11 == 0 || n % 11 == 1)
                b = true;
        }
			
		return b;
		
	}
	
	public static void main(String[] args) {
		
		logic1 o = new logic1();
		
		System.out.println(o.specialEleven(22));
		System.out.println(o.specialEleven(23));
		System.out.println(o.specialEleven(24));

	}

}
